central/signatureintegration/datastore/defaults.go

```diff
@@ -0,0 +1,14 @@
+package datastore
+
+import (
+	rekorClient "github.com/sigstore/rekor/pkg/generated/client"
+	"github.com/stackrox/rox/generated/storage"
+)
+
+func applyDefaultValues(integration *storage.SignatureIntegration) {
+	if tlog := integration.GetTransparencyLog(); tlog.GetEnabled() {
```

Comment:
nit: conditions could be combined with `&&`
```suggestion
	if tlog := integration.GetTransparencyLog(); tlog.GetEnabled() && tlog.GetRekorUrl() == "" {
```